lib/motion.h.rb

```diff
@@ -60,16 +60,15 @@ def verify_header_file
   def generate_bridgesupport_file
     return if File.exist?(bridgesupport_file)
     Dir.mkdir(@bridgesupport_dir) unless Dir.exist?(@bridgesupport_dir)
-    flag = begin
-      case platform
-      when :ios
-        "--no-64-bit"
-      when :osx
-        "--64-bit"
-      end
+    cflags = [
+      "-I#{include_path}",
+      "-F#{frameworks_path}"
+    ]
+    if `sw_vers -productVersion` =~ /^10.15/ # Catalina
```

Comment:
This fails for Big Sur. How about?

Gem::Version.new(`sw_vers -productVersion`) >= Gem::Version.new('10.15')